apps/pie-docs/src/designers/contributing/design-checklist.json

```diff
@@ -4,13 +4,13 @@
             "headingText": "Design for scalability and flexibility",
             "imgSrc": "../../assets/img/designers/contributing/checklist-scalability.svg",
             "imgSrcNarrow": "../../assets/img/designers/contributing/checklist-scalability_narrow.svg",
-            "text": "Ensure the component is responsive, flexible and is future proof. Think about all the places where it might be used, when and what for would designers use it, and design your new component accordingly."
+            "text": "Ensure the component is responsive, flexible and future-proof. Think about all the places it might be used - when, why  and how - and design your new component accordingly."
```

Comment:
Nit: double space
```suggestion
            "text": "Ensure the component is responsive, flexible and future-proof. Think about all the places it might be used - when, why and how - and design your new component accordingly."
```

Reply:
just pushed a change for this one. ta!